core/vm/evm.go

```diff
@@ -53,6 +53,13 @@ func (evm *EVM) precompile(addr common.Address) (PrecompiledContract, bool) {
 		precompiles = PrecompiledContractsHomestead
 	}
 	p, ok := precompiles[addr]
+	// Restrict overrides to known precompiles
+	if evm.chainConfig.IsOptimism() && evm.Config.OptimismPrecompileOverrides != nil {
```

Comment:
If we're restricting to only known precompiles should this be 
```suggestion
	// Restrict overrides to known precompiles
	if ok && evm.chainConfig.IsOptimism() && evm.Config.OptimismPrecompileOverrides != nil {
```

Reply:
Thanks for reading the comment!